lib/linguist/languages.yml

```diff
@@ -3736,6 +3736,15 @@ RenderScript:
   tm_scope: none
   ace_mode: text
   language_id: 323
+Ring:
+  type: programming
+  color: "#0e60e3"
+  extensions:
+  - .ring
+  - .rh
```

Comment:
@pchaigno Thanks, The search results
(1) https://github.com/search?p=100&q=extension%3Aring+see&ref=searchresults&type=Code&utf8=%E2%9C%93
(2) https://github.com/search?utf8=%E2%9C%93&q=*.ring&type=
(3) https://github.com/search?utf8=%E2%9C%93&q=*.rh&type=